### PR TITLE
Updates. DASH-188 Streamline registration and purchase flow

### PR DESCRIPTION
**Pull request by trukhilio**
- Remove pin and account confirmation from create account flow
- Add confirm email field
- Add infobage to create account form
- Remove email verification, email block page, update change email flow
- Check for pin in settings, update change pin content, make check pin in redux
- Update pin confirm - add password confirm
- Create PIN page
- Remove unused pin form
- Handle redirect for create account pin setup
- Set onchange validation for email field in create account
- Update success modal for recovery questions

### CODE REVIEW

File: client/src/pages/CreateAccountSetupPinPage/CreateAccountSetupPinPage.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import { PinConfirmComponent } from './components/PinConfirmComponent';
+
+import { setDataMutator } from '../../utils';
+
+import { CONFIRM_PIN_ACTIONS } from '../../constants/common';
+
+import { useContext } from './CreateAccountSetupPinPageContext';
+
+import classes from './CreateAccountSetupPinPage.module.scss';
+import BackButton from '../../components/common/BackButton/BackButton';
```
Comment (vmkarpenko) [inline client/src/pages/CreateAccountSetupPinPage/CreateAccountSetupPinPage.tsx:18]: import order

File: server/src/services/users/UpdateEmail.mjs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      });
+    }
+
+    await user.update({ email: newEmail });
+
+    return { data: { success: true } };
+  }
+
+  static get paramsSecret() {
+    return ['data.id'];
```
Comment (vmkarpenko) [inline server/src/services/users/UpdateEmail.mjs:37]: there is no such param like id

**Review by vmkarpenko — APPROVED** [review]

File: client/src/pages/CreateAccountSetupPinPage/CreateAccountSetupPinPage.tsx (hunk 1 above)
Comment (trukhilio) [reply client/src/pages/CreateAccountSetupPinPage/CreateAccountSetupPinPage.tsx]: Fixed

File: server/src/services/users/UpdateEmail.mjs (hunk 2 above)
Comment (trukhilio) [reply server/src/services/users/UpdateEmail.mjs]: Fixed

**Pull request merged**